subsys/net/ip/ipv6_nbr.c

```diff
@@ -82,7 +82,7 @@ static void ipv6_nd_restart_reachable_timer(struct net_nbr *nbr, int64_t time);
 #define DELAY_FIRST_PROBE_TIME (5 * MSEC_PER_SEC)
```

Comment:
About the commit `net/ip: Removing useless extra data capability in neighbors`, if it is not needed, then lets remove it. I just wonder what functionality used it and was removed (my guess is the RPL code what was removed eons ago).

Reply:
It was yes, as it was a port of contiki's. And actually, that way would be sub-optimal for zephyr which can handle multiple L2 and net iface at the same time which contiki cannot. So one net iface could be running rpl while all other won't, but all nbr would be affected in size by it.